lang/da.js

```diff
@@ -91,9 +91,9 @@ export default {
 	"components.more-less.less": "færre",
 	"components.more-less.more": "flere",
 	"components.overflow-group.moreActions": "Flere handlinger",
-	"components.pager-load-more.action": "Load {count} More",
-	"components.pager-load-more.info": "{showingCount} of {totalCount} items",
-	"components.pager-load-more.status-loading": "Loading more items",
+	"components.pager-load-more.action": "Indlæs {count} mere",
```

Comment:
A lot of these changed the casing on "More" -> "more". This might be a language-specific thing, so I'm tempted to just leave it.